### DISCUSSION TITLE
"Connection Errored Out" with Juggernaut XL Base model

### DISCUSSION THREAD

**Discussion by NeuUsir** [lllyasviel/Fooocus#3518] [Q&A]
Hello. I am a newbie. 
I started having problems like - Latency (the whole Device), Connection randomly timing out, Extremely slow Image generation etc immediately after I started using Juggernaut XL. I wonder if I messed up the Directory with this step. Any help would be greatly appreciated. Thank you. 
![1](https://github.com/user-attachments/assets/5a12f02a-c0c2-45d3-88b4-e487ff8b2f42)
![2](https://github.com/user-attachments/assets/33e9a91f-745c-4e05-ac33-080121cfd85d)
![3](https://github.com/user-attachments/assets/92152eaa-41f8-42e0-afe8-231df1c6b4da)
![4](https://github.com/user-attachments/assets/524eead2-9d7f-44ea-a2b3-a4a5b59890c0)

**Comment by Rogala** [comment]
Tested the model with different LORA and Quality Settings, no malfunctions, look for the problem in your program settings, maybe you changed something in the files.
![image](https://github.com/user-attachments/assets/b249b9e5-af69-42e4-bd9e-85376dcde2fa)
